doc/tutorials/custom-builds.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Custom builds
+layout: doc.hbs
+---
+
+# Creating custom builds
+
+OpenLayers 3 is a big library providing a lot of functionality. So it is unlikely that an application need and use all the functionality OpenLayers 3 provides. This is why creating application-specific OpenLayers 3 builds, with just the functionality your application needs, is often a good idea.
+
+An alternative to creating custom builds is to compile the application code and the OpenLayers 3 code. See the [Compiling Application with Closure Compiler](closure.html) tutorial for more information.
```

Comment:
Agreed.  Or it could read "An alternative ... is to compile your application code together with the library."